bin/ch-test

```diff
@@ -186,18 +337,35 @@ fi
 
 # Ensure BATS_TMPDIR is set to /tmp (issue #278).
 if [[ -n $BATS_TMPDIR ]]; then
-    if [[ ! $BATS_TMPDIR == '/tmp/'* ]]; then
+    if [[ "$BATS_TMPDIR" != '/tmp'* ]]; then
         fatal "BATS_TMPDIR: must be /tmp but found $BATS_TMPDIR (issue #278)"
     fi
 fi
 
+# Create a directory in user space to hold auto-generated test artifacts
```

Comment:
(1) Say "home directory" rather than "user space", which is a kernel concept.
(2) I think this belongs better in `$BATS_TMPDIR` rather than the home directory.
(3) What's the reasoning to keep the directory rather than deleting it?
(4) Note that `$BATS_TMPDIR` gets twiddled in `common.bash`. I wonder if that logic is better here?
(5) You'll want to make sure you have a fresh directory without any leftovers from a previous run.